docs/encryption.md

```diff
@@ -0,0 +1,19 @@
+# Export Encryption Compliance
+
+Your app must abide by [U.S. export restrictions](https://www.bis.doc.gov/index.php/policy-guidance/encryption) if it is available for download outside of the U.S. or Canada. This is because both Apple and Google's app store servers are in the United States. If a user from another country downloads the app, it is from a U.S. server. Thus, the server exports the app to another country.
```

Comment:
```suggestion
Your app must abide by [U.S. export restrictions](https://www.bis.doc.gov/index.php/policy-guidance/encryption) if it's available for download outside of the U.S. or Canada. This is because both Apple and Google's app store servers are in the United States. If a user from another country downloads the app, it comes from a U.S. server and the server exports the app to another country.
```